docs/snippets/ru/github-action-failing.md

```diff
@@ -0,0 +1,23 @@
+## GitHub Actions не выполняются
+
+### TLDR;
+
+Спасибо Вам за PR. Произошел сбой GitHub Action, пожалуйста проверьте Action output чтобы узнать больше об ошибке.
+
+По любым вопросам обращайтесь в Discord EddieHub http://discord.eddiehub.org
```

Comment:
Maybe consider using the link feature of markdown like `[test](test.com)`

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Thanks Tom 👍 , I think this could be another green square for someone else